kolibri/core/assets/src/utils/intl-locale-data.js

```diff
@@ -8,6 +8,12 @@
  */
 module.exports = function(locale) {
   switch (locale) {
+    case 'ach-ug':
```

Comment:
I think we should not add this - it's a special case and there is no intl locale polyfill for this. It will fallback to English I believe.

Reply:
thank you! I excluded `ach-ug` in the file `build_tools/i18n/intl_code_gen.js` so hopefully it won't be generate these lines in the future